source/openapi-spec.yml

```diff
@@ -58,6 +58,153 @@ paths:
                     type: array
                     items:
                       $ref: "#/components/schemas/Application"
+  "/applications/{application_id}/offer":
+    post:
+      summary: Make an offer
+      parameters:
+        - name: application_id
+          in: path
+          required: true
+          description: The unique ID of this application
+          schema:
+            type: string
+            maxLength: 10
+            example: 11fc0d3b2f
+      requestBody:
+        description: |
+          The conditions of the offer
+
+          Providing a request body with an array of conditions is equivalent to making a **conditional** offer. For an **unconditional** offer, provide an empty request body.
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
```

Comment:
Would it be appropriate to use the `#/components/schemas/Offer` ref here?

Reply:
Most likely! 👍 Just getting the endpoints created and then planning on refactoring and getting into the finer details after.

Reply:
👍 Great approach

Reply:
Yush, it can be done and has been done! 😉  - https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training-tech-docs/pull/69/commits/f982b29c6f29f7eb715a7350ddd1eb0da7066b65.